nsv13/code/modules/overmap/knpc.dm

```diff
@@ -61,6 +65,7 @@ GLOBAL_LIST_EMPTY(knpcs)
 /datum/component/knpc/Initialize()
 	if(!iscarbon(parent))
 		return COMPONENT_INCOMPATIBLE
+	failsafe_trust = rand((2 SECONDS), (5 SECONDS))
```

Comment:
Why did we define it if it's changed here? Maybe you can incorporate the value into the calculation or let setting it be an override for this (so people can make subtypes with different values easily)

Reply:
Uhhh good question.
But yeah makes sense.